### PR TITLE
Feat/gh13 shape positioning; fixes #12 #13 #14

### LINKED ISSUE #12: DST-CAT-SR-0080: Identify the Reference ElementDefinition
The software shall let the user identify the ElementDefinition that will act as a reference for the top-level coordinate system of the product of interest (i.e. the product that contains the datum).

### PR DESCRIPTION
**Pull request by nathanatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/DEH-CATIA/pulls) open
- [x] I have verified that I am following the DEH-CATIA [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/DEH-CATIA/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
fixes #12  #13  #14 

- [x] Allowing selecting the top element definition
- [x] Auto Creation/Update of element definition and usages
- [x] Creation/Update of parameters (orientation, position, volume, mass, CoG, MoI)

### CODE REVIEW

File: DEHCATIA/Services/ComConnector/CatiaComService.cs [hunk 1]
```diff
@@ -369,47 +446,42 @@ private IEnumerable<IDstParameterViewModel> ParseParameters(Parameters parameter
...
-        /// read the 'Analyze' object and calls the according function with it.
-        /// </summary>
-        /// <param name="partDocument">The CATIA <see cref="PartDocument"/> referring to the geometric top element</param>
-        /// <param name="element">The <see cref="ElementRowViewModel"/></param>
-        private void AddInertiaParameters(PartDocument partDocument, ElementRowViewModel element)
-        {
-            this.AddInertiaParameters(partDocument.Product.Analyze, element);
+            element.Shape = shape;
+
+            //element.Shape = new CatiaShapeViewModel()
```
Comment (alexatrhea) [inline DEHCATIA/Services/ComConnector/CatiaComService.cs:480]: delete commented code.

**Review by alexatrhea — APPROVED** [review]
approved with minor comment. Also for CATIA since we do not have a CI/CD solution yet, can you nad @ahmedatrhea please post coverage results directly from VS into the PR so we can control it?

**Comment by nathanatrhea** [comment]
image

**Review by ahmedatrhea — APPROVED** [review]

**Comment by alexatrhea** [comment]
@nathanatrhea you can exclude .Test project from coverage in ReSharper.

**Pull request merged**